sdk/core/core/src/az_http_policy.c

```diff
@@ -37,6 +32,14 @@ AZ_NODISCARD AZ_INLINE az_result az_http_pipeline_nextpolicy(
   return p_policies[0].pfnc_process(&(p_policies[1]), p_policies[0].data, hrb, response);
 }
 
+// Non-inlinable version to create function pointers to
+static AZ_NODISCARD az_result _az_http_pipeline_nextpolicy(
+    az_http_policy * const p_policies,
+    az_http_request_builder * const hrb,
+    az_http_response * const response) {
+  return az_http_pipeline_nextpolicy(p_policies, hrb, response);
```

Comment:
az_http_pipeline_nextpolicy is an internal function.  It should be named with a leading _